templates/template.md.jinja

```diff
@@ -24,36 +25,43 @@
 
 **Classes:** 
 {% for class in classes %}
-- [{{class.label}}]({{class.iri}})
+- [{{class.label}}](#{{class.shortname|lower|urlencode}})
 {% endfor %}
 
-**Properties:** 
+**Properties:**
+
 {% for property in properties %}
-- [{{property.label}}]({{property.iri}})
+- [{{property.label}}](#{{property.shortname|lower|urlencode}})
 {% endfor %}
 
 {% for class in classes %}
-## Class: [{{class.label}}]({{class.iri}})
+## <a name="{{class.shortname|lower|urlencode}}"> Class: {{class.label}} </a>
+**{{class.shortname}}** <br>
+{{class.iri}} <br>
+{{class.description}}
 
+
+{% if class.parent is defined and class.parent|length > 0 %}
 **Subclass of:** 
-{% if class.parent is defined %}
 {% for parent in class.parent %}
 [{{parent.label}}]({{parent.iri}}), 
 {% endfor %}
 {% endif %}
+{% if class.child is defined and class.child|length > 0%}
 **Subclasses:** 
-{% if class.child is defined %}
 {% for child in class.child %}
 [{{child.label}}]({{child.iri}}), 
 {% endfor %}
 {% endif %}
 
 | Property | URI   | Description | Datatype | Min | Max |
```

Comment:
Eigenlijk zou je hier "automatisch" de properties van de parent moeten meenemen, eventueel met een aanduiding dat ze van een andere klasse overgeërfd zijn

Reply:
Van alle voorafgaande parents? Dus ook parent van parent enz... ?

Reply:
liefst wel, maar mss op een manier dat je het kan configureren wel/niet?

Reply:
En om het niet te onoverzichtelijk te maken, mss het doen zoals javadoc. Zie bv.  https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/ext/xerces/jaxp/datatype/DatatypeFactoryImpl.html en "Methods inherited from class ..."

Reply:
ik zou wel het woord "inherited" niet gebruiken want het is niet echt correct en het doet de mensen te veel aan OO denken.